src/api/userApi.js

```diff
@@ -3,6 +3,13 @@ import { get, post, del, handleResponseAndHeaders } from './apiUtils';
 export const signIn = async credentials =>
   await post('/users/sign_in', { user: credentials }, { handleSuccess: handleResponseAndHeaders });
 
+export const facebookSignIn = async token =>
+  await post(
+    '/users/facebook',
+    { access_token: token },
```

Comment:
In javascript the standard is to use camelCase so here it should be
```suggestion
    { accessToken: token },
```
To do so, you need to convert from camel case to underscore the data sent to backend. 
For this you can use `decamelize` from humps (https://www.npmjs.com/package/humps) in your `apiRestCall` method in `apiUtils.js`:

`body: body && JSON.stringify(humps.decamelizeKeys(body))`

Reply:
And you can use `camelize` too for the data you get from backend in , to convert it from underscore to camel case, in `handleResponse` method.

Reply:
I'll create a new PR to apply these changes!